using-valtimo/plugin/public-task/configure-public-task-plugin.md

```diff
@@ -0,0 +1,58 @@
+# Public Task Plugin
```

Comment:
Formatting is messed up around the images. Please make use of newlines in order to break everything up correctly.